### ISSUE TITLE
Monster Info

### ISSUE THREAD

**Issue by Pyromunky** [runelite/runelite#4004]
Would it be possible to have a lookup option on monsters that make the side menu pop up with the monster stats and drop table? This way it stays in client?

**Comment by deathbeam** [comment]
I dont think that makes a lot of sense. Only reason for that would be to not require opening browser and that is quite horrid reason. But, here is PR that adds lookup option and opens wiki page for you what I like a lot more: https://github.com/runelite/runelite/pull/2829
